moveit_core/robot_state/test/robot_state_benchmark.cpp

```diff
@@ -38,8 +38,8 @@
 #include <moveit/robot_state/robot_state.h>
 #include <moveit/utils/robot_model_test_utils.h>
 #include <eigen_stl_containers/eigen_stl_containers.h>
-#include <gtest/gtest.h>
 #include <chrono>
+#include <gtest/gtest.h>
```

Comment:
How are the changes in this file related to the port?

Reply:
it's not part of the port (code-wise) but I had to modify the CMakelists.txt with `ament_add_gtest`. So I used this opportunity to also fix the tests.